node/errors.go

```diff
@@ -141,3 +141,6 @@ var ErrNilCreateTransactionArgs = errors.New("nil args for create transaction")
 
 // ErrNilNodeRunner signals that a nil node runner was provided
 var ErrNilNodeRunner = errors.New("nil node runner")
+
+// ErrNilNode signals that a nil node was provided
+var ErrNilNode = errors.New("nil node")
```

Comment:
This error is already defined in facade/errors.go
Please do not duplicate it. Cleanest solution would be to move the error from facade package to `errors/errors.go`

Reply:
moved ErrNilNode to `errors/errors.go`